types/mini-css-extract-plugin/index.d.ts

```diff
@@ -6,7 +6,8 @@
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Configuration, Compiler } from 'webpack';
+
+import { Configuration, ChunkData, Plugin, Compiler } from 'webpack';
```

Comment:
why this specific imports? It's clear in case of `compression-webpack-plugin1, but not here, thx!

Reply:
Hi

It resulted in a typo from using the github.com online resolver thing for the first time. I've removed the redundant imports. I opted to leave the `Compiler` import there because it's a parameter of the `apply` function